avae/data.py

```diff
@@ -296,8 +302,12 @@ def read(self, filename):
 
     def voxel_transformation(self, x):
 
+        if self.rescale is not None:
```

Comment:
Shouldn't this be `if self.rescale`, this will be applied even if self.rescale=False as is given in the default config. Also, for the shrec dataset lines 306 and 307 are creating an empty `x` array.